.github/workflows/gradle-build.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ dev, master ]
```

Comment:
should probably salvage the exclude from the other pipeline 
https://github.com/microsoftgraph/msgraph-sdk-java-core/blob/54ae401d342a07eef337086f63cca1da5a79d59c/.azure-pipelines/buildAndPackage.yml#L15

Reply:
add workflow dispatch so we can queue things manually